homeassistant/components/romy/config_flow.py

```diff
@@ -0,0 +1,111 @@
+"""Config flow for ROMY integration."""
+from __future__ import annotations
+
+import romy
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.components import zeroconf
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PASSWORD
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+
+from .const import DOMAIN, LOGGER
+
+
+def _schema_with_defaults(
+    host: str = "", name: str = "", requires_password: bool = False
+) -> vol.Schema:
+    schema = {
+        vol.Required(CONF_HOST, default=host): cv.string,
+        vol.Optional(CONF_NAME, default=name): cv.string,
+    }
+
+    if requires_password:
+        schema.update(
+            {vol.Required(CONF_PASSWORD, default=""): vol.All(str, vol.Length(8))}
+        )
```

Comment:
```suggestion
        schema[vol.Required(CONF_PASSWORD)]= vol.All(str, vol.Length(8))
```

Don't set an invalid default value